docs/data/sdks/android-kotlin/index.md

```diff
@@ -293,3 +293,494 @@ amplitude.add(
     new SegmentDestinationPlugin(this, SEGMENT_WRITE_KEY)
 )
 ```
+
+## Advanced Topic
+
+### User sessions
+
+A session on Android is a period of time that a user has the app in the foreground.
+
+Amplitude groups events together by session. Events that are logged within the same session have the same `session_id`. Sessions are handled automatically so you don't have to manually call `startSession()` or `endSession()`.
+
+You can adjust the time window for which sessions are extended.
+
+=== "Java"
+
+    ```
+    amplitude = AmplitudeKt.Amplitude(AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setMinTimeBetweenSessionsMillis(1000);
+        return Unit.INSTANCE;
+    });
+    ```
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+            Configuration(
+                apiKey = AMPLITUDE_API_KEY,
+                context = applicationContext,
+                minTimeBetweenSessionsMillis = 10000
+            )
+        )
+    ```
+
+By default, Amplitude will auto send the '[Amplitude] Start Session' and '[Amplitude] End Session'. Even though these events aren't sent, sessions are still tracked by using `session_id`.
+You can also disable those session events.
+
+=== "Java"
+
+    ```
+    amplitude = AmplitudeKt.Amplitude(AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setTrackingSessionEvents(false);
+        return Unit.INSTANCE;
+    });
+    ```
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+            Configuration(
+                apiKey = AMPLITUDE_API_KEY,
+                context = applicationContext,
+                trackingSessionEvents = false
+            )
+        )
+    ```
+
+You can use the helper method getSessionId to get the value of the current sessionId.
+
+=== "Java"
+
+    ```java
+    long sessionId = amplitude.getSessionId();
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    val sessionId = amplitude.sessionId;
+    ```
+
+!!!note
+    For Android API level 14 and above, a new session is created when the app comes back into the foreground after being in the background for five or more minutes or when the last event was logged
```

Comment:
```suggestion
    For Android API level 14 and higher, a new session is created when the app comes back into the foreground after being in the background for more than five minutes or when the last event was logged.
```

Clarifying question -- so a new session is created after 5 minutes of the app being backgrounded OR after 5 minutes pass since the last logged event?

Reply:
I just noticed we have different behavior in the new android SDK. I will change some words here. 
It should be pass x minutes since the last logged event.